src/components/Employee/Deductions/IncludeDeductionsForm/IncludeDeductionsFormEmptyState.module.scss

```
@@ -0,0 +1,22 @@
+.emptyStateContainer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 411px;
```

this was just pulled from the figma spec

---

You should look and figure out what variable this most aligns to, I feel like we might have defined this before

---

Would also recommend using the `toRem` utility which should be available in the global scope. I also tend to round down a pixel in cases like this when the number is especially ugly but that's optional 😂 

```suggestion
  min-height: toRem(410);
```

---

I checked `src/contexts/ThemeProvider/theme.ts` but there wasn't  any defined variables for `min-height` -- is there any other files I should be checking? I looked through the base scss files but didnt find any thing similar either

---

Hmm, yeah I was looking at EmptyData.tsx and its styles and it doesn't look like we've ever defined Min height for those either.

This is probably ok for now but I'd check with Steve, our theme master